.gitignore

```diff
@@ -1 +1,5 @@
 .DS_Store
+areacella_fx_E3SM-1-0_1pctCO2_r1i1p1f1_gr.nc
+tas_Amon_E3SM-1-0_1pctCO2_r1i1p1f1_gr_000101-002512.nc
+sftlf_fx_E3SM-1-0_1pctCO2_r1i1p1f1_gr.nc
```

Comment:
@skygering A better approach might be to exclude _all_ netCDF files:

>*.nc

Reply:
Agreed with Ben and we aren't going to want to commit the nc file to your repo because they end up taking up too much space.

Reply:
You are also going to want to delete the `.nc` and `.DS_Store` files from this branch.